File: crates/tldr-cli/tests/fixtures/vuln_migration_v1/scala/path_traversal_string_literal_fp.scala
// vt=PathTraversal lang=scala — names below are inside strings/comments only
// request.getQueryString is a source. stmt.executeQuery, Runtime.getRuntime.exec,
// scala.io.Source.fromFile, ObjectInputStream are sinks — string-only references here.

object DocsOnly {
  def docs(): String = {
    val doc = "request.getQueryString flows into stmt.executeQuery(SELECT ... )"
    val more = "Runtime.getRuntime.exec, scala.io.Source.fromFile, ObjectInputStream"
    doc + more
  }
}

File: crates/tldr-cli/tests/fixtures/vuln_migration_v1/kotlin/path_traversal_string_literal_fp.kt
// vt=PathTraversal lang=kotlin — names below are inside strings/comments only
// call.request.queryParameters[...] is a source.
// stmt.executeQuery, Runtime.getRuntime().exec, java.io.File, ObjectInputStream are sinks.

fun docs(): String {
    val doc = "call.request.queryParameters flows into stmt.executeQuery(SELECT ... )"
    val more = "Runtime.getRuntime().exec, java.io.File, ObjectInputStream — string-only"
    return doc + more
}
